.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -301,10 +301,17 @@ jobs:
       - name: Post-test information gathering
         if: ${{ !success() }}
         run: |
-          cilium status
+          echo "=== Install latest stable CLI ==="
+          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz{,.sha256sum}
+          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
+          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/bin
+          rm cilium-linux-amd64.tar.gz{,.sha256sum}
+          cilium version
```

Comment:
This part should not be needed since we already install the CLI as part of the `cilium/cilium` workflows :)

Reply:
I figured we'd want the latest CLI as in cilium/cilium-cli, to quickly benefit from any sysdump improvement. I'm even tempted to use master instead of latest stable. Is there any benefit to tying the version used for the tests and the version used for the sysdump?

Reply:
> I figured we'd want the latest CLI as in cilium/cilium-cli

Not sure if that was what you meant so disregard if irrelevant, but for context regarding why `cilium/cilium-cli` manually downloads the latest CLI: it is because we explicitly don't want to use the PR-built CLI and have it run in a `pull_request_target` privileged context (more details on https://github.com/cilium/cilium-cli/pull/423).

> Is there any benefit to tying the version used for the tests and the version used for the sysdump?

I guess same as pinning CLI verison rather than using `latest` or `master`: ensure that it works in a consistent and controlled manner.

Reply:
Ok, I'm not strongly convinced either way, so let's go for the simplest. I've removed the additional code such that we're using the same CLI version everywhere.